docs/requirements.txt

```diff
@@ -1,26 +1,7 @@
-alabaster==0.7.12
-Babel==2.9.1
-certifi==2019.11.28
-chardet==3.0.4
-colorama==0.4.3
-docutils==0.16
-idna==2.9
-imagesize==1.2.0
-Jinja2==2.11.3
-MarkupSafe==1.1.1
-packaging==20.3
-Pygments==2.7.4
-pyparsing==2.4.6
-pytz==2019.3
-requests==2.23.0
-six==1.14.0
-snowballstemmer==2.0.0
-Sphinx==2.4.4
-sphinx-rtd-theme==0.4.3
-sphinxcontrib-applehelp==1.0.2
-sphinxcontrib-devhelp==1.0.2
-sphinxcontrib-htmlhelp==1.0.3
-sphinxcontrib-jsmath==1.0.1
-sphinxcontrib-qthelp==1.0.3
-sphinxcontrib-serializinghtml==1.1.4
-urllib3==1.26.5
+Sphinx>=7.0.0
```

Comment:
Use these version constraints in the requirement file: https://github.com/nexB/skeleton/blob/main/setup.cfg#L60

Reply:
alright
are there some compatibility issues with later versions ?

Reply:
Updated